tests/system/marathon_pods_tests.py

```diff
@@ -454,7 +454,12 @@ def test_pod_with_container_bridge_network():
     assert agent_ip != container_ip, "The container IP address is the same as the agent one"
 
     url = "http://{}:{}/".format(agent_ip, port)
-    common.assert_http_code(url)
+
+    @retrying.retry(wait_fixed=1000, stop_max_attempt_number=60, retry_on_exception=common.ignore_exception)
```

Comment:
So the problem was that nginx hasn't fully booted up yet?

Reply:
Maybe :)